Bottom: 891a16cb25ad39db51200ed1eb3262b8f982b9bd
Top:    7d9e5cc230e2810cf4b2e08f71e5100c4d4d5941
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-16 15:58:18 -0700

Refresh of rs-delay

---

diff --git a/examples/rstream.c b/examples/rstream.c
index 825c4de..a214b1a 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -485,7 +485,6 @@ static int set_test_opt(char *optarg)
 			break;
 		case 'b':
 			flags &= ~MSG_DONTWAIT;
-			no_delay = 0;
 			break;
 		case 'n':
 			flags |= MSG_DONTWAIT;
@@ -503,7 +502,6 @@ static int set_test_opt(char *optarg)
 			use_async = 1;
 		} else if (!strncasecmp("block", optarg, 5)) {
 			flags &= ~MSG_DONTWAIT;
-			no_delay = 0;
 		} else if (!strncasecmp("nonblock", optarg, 8)) {
 			flags |= MSG_DONTWAIT;
 		} else if (!strncasecmp("verify", optarg, 6)) {
